src/config/support-page.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-return [
-    'enquiry_url' => 'https://systems.hiav.networkrail.co.uk/home/forms/enquiry/start',
+use Spatie\Permission\Models\Role;
 
+return [
     'excluded_roles' => [],
 
     'permission' => 'manage_support_page',
 
-    'role_model' => Spatie\Permission\Models\Role::class,
+    'role_model' => Role::class,
```

Comment:
Either use FQN or Import statements consistently. Don't mix them.

Reply:
No problem! Should I change user model to be an import?

Reply:
Thanks, I will put it back to the FQN, I got confused with the comment on the template and applied it here, when it should be on template